src/hooks/useLoggedIn.js

```diff
@@ -4,9 +4,11 @@ import 'firebase/auth';
 
 export const useLoggedIn = () => {
   const [user, setUser] = useState({ loggedIn: false });
+  const [loaded, setLoaded] = useState({ loaded: false });
```

Comment:
Does this value need to be an object? And tiny thing: can this be `isLoaded` and `setIsLoaded` 😬

Reply:
Yea I feel like this is where typescript would help out, but maybe making it `authIsLoaded` would be good since there's a lot of 'loaded's' flying around and it gets renamed to authIsLoaded anyways.

Making the value an object keeps it consistent with the current return object, which is looking like:

```
{
  loggedIn: <boolean>,
  user: <user>,
  authIsLoaded: <boolean>,
}
```

Reply:
💯